ChatDemo-UI3.0/ChatDemo-UI3.0/en.lproj/Localizable.strings

```diff
@@ -316,12 +316,12 @@
 "setting.inputName" = "Please input nickname";
 "setting.saveName" = "Save Nickname";
 "setting.namenotempty" = "Name cannot be empty";
-"setting.personalInfo" = "Profile";
+"setting.personalInfo" = "User Profile";
 "setting.setBitrate" = "Set Bitrate";
 "setting.setBitrateTips" = "Set Bitrate should be 150-1000";
 "setting.personalInfoUpload" = "Profile Image";
 "setting.edittips" = "After setting this nickname, chat with the iOS client demo project, iOS will display this nickname is not a EaseMob ID, if the other party to use the Android client this setting is not effective";
-"setting.iospushname" = "iOS push nickname";
+"setting.iospushname" = "APNs Display Name";
```

Comment:
ok, good idea. 

Jerry






> On May 27, 2016, at 10:46 AM, weapon3042 <notifications@github.com> wrote:
> 
> In ChatDemo-UI3.0/ChatDemo-UI3.0/en.lproj/Localizable.strings <https://github.com/easemob/sdkdemoapp3.0_ios/pull/47#discussion_r64942394>:
> 
> >  "setting.setBitrate" = "Set Bitrate";
> >  "setting.setBitrateTips" = "Set Bitrate should be 150-1000";
> >  "setting.personalInfoUpload" = "Profile Image";
> >  "setting.edittips" = "After setting this nickname, chat with the iOS client demo project, iOS will display this nickname is not a EaseMob ID, if the other party to use the Android client this setting is not effective";
> > -"setting.iospushname" = "iOS push nickname";
> > +"setting.iospushname" = "APNs Display Name";
> Since you changed APNS settings to Push Notification, I assume Notification Display Name is better here, APNS is not necessarily known by everyone
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/easemob/sdkdemoapp3.0_ios/pull/47/files/41aa668f908377e0f353499e5e51c13aea8eb0aa#r64942394>, or mute the thread <https://github.com/notifications/unsubscribe/AAeKCEQ2Y78w3zt89Zd7rHcdXKdxnoGFks5qFy4GgaJpZM4IoNPI>.
>